migrations/20220505165533_create_table_posts.sql

```diff
@@ -0,0 +1,13 @@
+-- Add migration script here
+
+CREATE TYPE scope as ENUM ('public', 'private');
+
+CREATE TABLE IF NOT EXISTS posts (
+	id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
+	user_id UUID NOT NULL,
+	content VARCHAR(320) NOT NULL,
+	scope scope NOT NULL,
+	created_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+	updated_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+	FOREIGN KEY (user_id) REFERENCES users(id)
+)
```

Comment:
```suggestion
-- Add migration script here
CREATE TYPE scope as ENUM ('public', 'private');

CREATE TABLE IF NOT EXISTS posts (
  id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
  user_id UUID NOT NULL,
  content VARCHAR(320) NOT NULL,
  scope scope NOT NULL,
  created_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
  updated_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
  FOREIGN KEY (user_id) REFERENCES users(id)
)
```

We use 2 spaces over tabs.